synapseclient/client.py

```diff
@@ -246,7 +245,26 @@ def __init__(
         requests_session: requests.Session = None,
         cache_root_dir: str = None,
         silent: bool = None,
-    ):
+    ) -> "Synapse":
+        """
+        Initialize Synapse object
+
+        Arguments:
+            repoEndpoint:       Location of Synapse repository. Defaults to None.
+            authEndpoint:       Location of authentication service. Defaults to None.
+            fileHandleEndpoint: Location of file service. Defaults to None.
+            portalEndpoint:     Location of the website. Defaults to None.
+            debug:              Print debugging messages if True. Defaults to None.
+            skip_checks:        Skip version and endpoint checks. Defaults to False.
+            configPath:         Path to config File with setting for Synapse. Defaults to CONFIG_FILE.
+            requests_session:   A custom requests.Session object that this Synapse instance will use
```

Comment:
```suggestion
            requests_session:   A custom requests. Session object that this Synapse instance will use
```

Reply:
@BryanFauble I believe the `requests.Session` means the [Session object ](https://requests.readthedocs.io/en/latest/user/advanced/#:~:text=more%20advanced%20features.-,Session%20Objects,-%C2%B6)of the request library. How about I add a link here?

Reply:
Just a normal hyperlink:
```
[Text](link)
```

Reply:
A link to their docs sounds good though!